apps/admin-gui/src/app/users/pages/user-detail-page/user-settings/user-roles/user-roles.component.html

```diff
@@ -1,8 +1,341 @@
+        <perun-web-apps-vo-select-table
+          [displayedColumns]="['id', 'name', 'shortName']"
+          [vos]="vos">
+        </perun-web-apps-vo-select-table>
+      </div>
+    </div>
+  </mat-expansion-panel>
+
+  <mat-expansion-panel (opened)="getAdminVos()" *ngIf="isVoAdmin">
+    <mat-expansion-panel-header>
+      <mat-panel-title>
+        <mat-icon mat-card-avatar svgIcon='perun-vo-black' class="mr-2"></mat-icon>
+        <p class="mt-auto mb-auto">{{'ROLES.VOADMIN' | translate}}</p>
+      </mat-panel-title>
+    </mat-expansion-panel-header>
+    <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+    <div *ngIf="!loading">
+      {{'ROLES.VOADMIN_VOS' | translate}}
+      <perun-web-apps-vo-select-table
+        [displayedColumns]="['id', 'name', 'shortName']"
+        [vos]="vos">
+      </perun-web-apps-vo-select-table>
+    </div>
+  </mat-expansion-panel>
+
+  <mat-expansion-panel (opened)="getVos(roles.get('VOOBSERVER').get('Vo'))" *ngIf="isVoObserver">
+    <mat-expansion-panel-header>
+      <mat-panel-title>
+        <mat-icon mat-card-avatar svgIcon='perun-vo-black' class="mr-2"></mat-icon>
+        <p class="mt-auto mb-auto">{{'ROLES.VOOBSERVER' | translate}}</p>
+      </mat-panel-title>
+    </mat-expansion-panel-header>
+    <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+    <div *ngIf="!loading">
+      {{'ROLES.VOOBSERVER_VOS' | translate}}
+      <perun-web-apps-vo-select-table
+        [displayedColumns]="['id', 'name', 'shortName']"
+        [vos]="vos">
+      </perun-web-apps-vo-select-table>
+    </div>
+  </mat-expansion-panel>
+
+  <mat-expansion-panel (opened)="getResourcesData('RESOURCEADMIN')" *ngIf="isResourceAdmin">
+    <mat-expansion-panel-header>
+      <mat-panel-title>
+        <mat-icon mat-card-avatar svgIcon='perun-resource-black' class="mr-2"></mat-icon>
+        <p class="mt-auto mb-auto">{{'ROLES.RESOURCEADMIN' | translate}}</p>
+      </mat-panel-title>
+    </mat-expansion-panel-header>
+    <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+    <div *ngIf="!loading">
+      {{'ROLES.RESOURCEADMIN_RESOURCES' | translate}}
+      <perun-web-apps-resources-list
+        [resources]="resources"
+        [routingVo]="true"
+        [displayedColumns]="['id', 'name', 'vo', 'facility', 'description']">
+      </perun-web-apps-resources-list>
+
+      <div class="mt-5">
+        {{'ROLES.RESOURCEADMIN_VOS' | translate}}
+        <perun-web-apps-vo-select-table
+          [displayedColumns]="['id', 'name', 'shortName']"
+          [vos]="vos">
+        </perun-web-apps-vo-select-table>
+      </div>
+
+      <div class="mt-5">
+        {{'ROLES.RESOURCEADMIN_FACILITIES' | translate}}
+        <app-facility-select-table
+          [displayedColumns]="['id', 'name']"
+          [checkAuth]="true"
+          [facilities]="facilities">
+        </app-facility-select-table>
+      </div>
+
+    </div>
+  </mat-expansion-panel>
```

Comment:
![image](https://user-images.githubusercontent.com/32871167/104292640-5cf45280-54bd-11eb-9c3e-09160e8a7c5e.png)
In the resource admin section also show only the table with resources(the table itself is showing the name of the VO and the name of the facility). The tables with facility and vo are redundant.